src/main/java/nl/novi/techiteasycontroller/controllers/CiModuleController.java

```diff
@@ -0,0 +1,58 @@
+package nl.novi.techiteasycontroller.controllers;
+
+import nl.novi.techiteasycontroller.dtos.CiModuleOutputDto;
+import nl.novi.techiteasycontroller.models.CiModule;
+import nl.novi.techiteasycontroller.service.CiModuleService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class CiModuleController {
+
+    private final CiModuleService ciModuleService;
+
+    public CiModuleController(CiModuleService ciModuleService) {
+        this.ciModuleService = ciModuleService;
+    }
+
+    @GetMapping("/ci-modules")
+    public ResponseEntity <List<CiModule>> getAllCiModules() {
+
+
+        List<CiModule> ciModules = this.ciModuleService.getAllCiModules();
+
+        return ResponseEntity.ok(ciModules);
+    }
+
+    @GetMapping("/ci-modules/{id}")
+    public ResponseEntity <CiModule> getCiModuleById(@PathVariable Long id) {
+
+        CiModule ciModule = this.ciModuleService.getCiModuleById(id);
+
+        return ResponseEntity.ok(ciModule);
+    }
+
+    @PostMapping("/ci-modules")
+    public ResponseEntity <CiModuleOutputDto> addCiModule(@RequestBody CiModuleOutputDto ciModuleOutputDto) {
+        CiModuleOutputDto ciModuleOutputDto1 = this.ciModuleService.addCiModule(ciModuleOutputDto);
+        return ResponseEntity.ok(ciModuleOutputDto1);
```

Comment:
.created is hier denk een betere optie dan .ok

Reply:
Sterker nog, created is hier de enige juiste optie